src/Event/Productions/ProductionCommandHandler.php

```diff
@@ -52,6 +61,45 @@ public function handleRemoveEventFromProduction(RemoveEventFromProduction $comma
     public function handleMergeProductions(MergeProductions $command): void
     {
         $toProduction = $this->productionRepository->find($command->getTo());
+        $fromProduction = $this->productionRepository->find($command->getFrom());
+
         $this->productionRepository->moveEvents($command->getFrom(), $toProduction);
+
+        $this->markNewMergedPairsAsLinked(
+            $fromProduction,
+            $toProduction
+        );
```

Comment:
Suggestion (also sent you this on Slack but posting it here for reference):
Maybe we should move this functionality of excluding the pairs to a decorator around `ProductionCommandHandler::handleMergeProductions()`?

Reply:
Or as an event listener reacting to `ProductionsWereMerged`?

Reply:
Ah, I didn't know/remember there was an event for that

Reply:
I would rather go with an event/event listener. But in any case, I would rather refactor this in later PRs because I would like to focus on implementing the rest of the API endpoints and integrating them with GCP asap

Reply:
Since there is actually not much endpoints left (one) I could also tackle this refactoring first :)

Reply:
Let's maybe tackle it in a follow-up so we can split up the work?